.circleci/config.yml

```diff
@@ -63,9 +63,9 @@ jobs:
           shell: bash.exe
           command: |
             python --version && \
-            pip install --upgrade pip setuptools wheel && \
-            pip install ".[dev-core,dev-civisml]" && \
-            pip list && \
+            python.exe -m pip install --upgrade pip setuptools wheel && \
+            python.exe -m pip install ".[dev-core,dev-civisml]" && \
+            python.exe -m pip list && \
```

Comment:
We never had an issue here before, but it's only now that we tripped over here when we didn't have `python.exe -m` 🤷